CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
+
+### Bug or Regression
+- Provides a patch to stop using urllib3 deprecated APIs until the upgrade to latest OpenAPI Generator (>6.4.0)
```

Comment:
CHANGELOG is automatically generated during release. this is not needed.

Reply:
ok, so just remove this change?

Reply:
Yes, please

Reply:
Done